include/sound/soc.h

```diff
@@ -1225,6 +1226,7 @@ struct soc_mixer_control {
 	unsigned int sign_bit;
 	unsigned int invert:1;
 	unsigned int autodisable:1;
+	unsigned int sdca_vol_q7p8:1;
```

Comment:
I am a little nervous about pulling this into the ASoC core, have you looked at adding wrappers for snd_soc_get_volsw/snd_soc_put_volsw in the SDCA code, if it doesn't end up looking too messy I feel like that might be preferrable.